dev/ese/published/inc/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+configure_file(os/cprintf.hxx ${INC_OUTPUT_DIRECTORY}/cprintf.hxx COPYONLY)
+configure_file(os/edbg.hxx ${INC_OUTPUT_DIRECTORY}/edbg.hxx COPYONLY)
+configure_file(os/encrypt.hxx ${INC_OUTPUT_DIRECTORY}/encrypt.hxx COPYONLY)
+configure_file(os/error.hxx ${INC_OUTPUT_DIRECTORY}/error.hxx COPYONLY)
+configure_file(os/EseEventTrace.g.hxx ${INC_OUTPUT_DIRECTORY}/EseEventTrace.g.hxx COPYONLY)
+configure_file(os/EseEventTraceType.g.hxx ${INC_OUTPUT_DIRECTORY}/EseEventTraceType.g.hxx COPYONLY)
+configure_file(os/event.hxx ${INC_OUTPUT_DIRECTORY}/event.hxx COPYONLY)
+configure_file(os/hapublish.hxx ${INC_OUTPUT_DIRECTORY}/hapublish.hxx COPYONLY)
+configure_file(os/library.hxx ${INC_OUTPUT_DIRECTORY}/library.hxx COPYONLY)
+configure_file(os/math.hxx ${INC_OUTPUT_DIRECTORY}/math.hxx COPYONLY)
+configure_file(os/memfile.hxx ${INC_OUTPUT_DIRECTORY}/memfile.hxx COPYONLY)
+configure_file(os/memory.hxx ${INC_OUTPUT_DIRECTORY}/memory.hxx COPYONLY)
+configure_file(os/memorynotification.hxx ${INC_OUTPUT_DIRECTORY}/memorynotification.hxx COPYONLY)
+configure_file(os/norm.hxx ${INC_OUTPUT_DIRECTORY}/norm.hxx COPYONLY)
+configure_file(os/osdiskapi.hxx ${INC_OUTPUT_DIRECTORY}/osdiskapi.hxx COPYONLY)
+configure_file(os/oseventtrace.hxx ${INC_OUTPUT_DIRECTORY}/oseventtrace.hxx COPYONLY)
+configure_file(os/oseventtrace.g.hxx ${INC_OUTPUT_DIRECTORY}/oseventtrace.g.hxx COPYONLY)
+configure_file(os/osfileapi.hxx ${INC_OUTPUT_DIRECTORY}/osfileapi.hxx COPYONLY)
+configure_file(os/osfs.hxx ${INC_OUTPUT_DIRECTORY}/osfs.hxx COPYONLY)
+configure_file(os/osfsconfig.hxx ${INC_OUTPUT_DIRECTORY}/osfsconfig.hxx COPYONLY)
+configure_file(os/osfsapi.hxx ${INC_OUTPUT_DIRECTORY}/osfsapi.hxx COPYONLY)
+configure_file(os/ostimerqueue.hxx ${INC_OUTPUT_DIRECTORY}/ostimerqueue.hxx COPYONLY)
+configure_file(os/perfmon.hxx ${INC_OUTPUT_DIRECTORY}/perfmon.hxx COPYONLY)
+configure_file(os/osdiag.hxx ${INC_OUTPUT_DIRECTORY}/osdiag.hxx COPYONLY)
+configure_file(os/string.hxx ${INC_OUTPUT_DIRECTORY}/string.hxx COPYONLY)
+configure_file(os/sysinfo.hxx ${INC_OUTPUT_DIRECTORY}/sysinfo.hxx COPYONLY)
+configure_file(os/task.hxx ${INC_OUTPUT_DIRECTORY}/task.hxx COPYONLY)
+configure_file(os/thread.hxx ${INC_OUTPUT_DIRECTORY}/thread.hxx COPYONLY)
+configure_file(os/time.hxx ${INC_OUTPUT_DIRECTORY}/time.hxx COPYONLY)
+configure_file(os/trace.hxx ${INC_OUTPUT_DIRECTORY}/trace.hxx COPYONLY)
+configure_file(os/types.hxx ${INC_OUTPUT_DIRECTORY}/types.hxx COPYONLY)
+configure_file(os/error.hxx ${INC_OUTPUT_DIRECTORY}/error.hxx COPYONLY)
+configure_file(os/error.hxx ${INC_OUTPUT_DIRECTORY}/error.hxx COPYONLY)
+configure_file(resmgr.hxx ${INC_OUTPUT_DIRECTORY}/resmgr.hxx COPYONLY)
+configure_file(stat.hxx ${INC_OUTPUT_DIRECTORY}/stat.hxx COPYONLY)
+configure_file(sync.hxx ${INC_OUTPUT_DIRECTORY}/sync.hxx COPYONLY)
+configure_file(tcconst.hxx ${INC_OUTPUT_DIRECTORY}/tcconst.hxx COPYONLY)
+configure_file(cc.hxx ${INC_OUTPUT_DIRECTORY}/cc.hxx COPYONLY)
```

Comment:
Also this one, listed above then two more times here  ... but anything worth doing once, is worth doing thrice I say!!! ;)